src/logger/simple.py

```diff
@@ -0,0 +1,121 @@
+import time
+from collections import defaultdict
+
+from loguru import logger
+from torch import Tensor
+from typing import Optional, Dict
+import wandb
+
+
+class Logger():
+    def __init__(self, print_freq: int = 100, image_freq: int = 1000, tb_path: str = None):
+        self.print_freq: int = print_freq
+        self.image_freq: int = image_freq
+        self.loss_buff: dict[str, dict[str, list]] = defaultdict()
+        self.loss_buff['values'] = defaultdict(list)
+        self.loss_buff['sumlast'] = defaultdict(int)
+        self.loss_buff['sum'] = defaultdict(int)
+        self.metrics_buff: dict[str, dict[str, list]] = defaultdict()
+        self.metrics_buff['values'] = defaultdict(list)
+        self.metrics_buff['sumlast'] = defaultdict(int)
+        self.metrics_buff['sum'] = defaultdict(int)
+        self.tb_path: str = tb_path
+        self.train_iter = 1
+        self.val_iter = 1
+        self.wandb = wandb.init(project="TextDeepFake", entity="sphericalpotatoinvacuum")
```

Comment:
There will be a mess due to different users -- we need to agree on something common for all, to init it somehow from user's machine

Reply:
project -- as a param in config, entity -- team name from Wanda